app/controllers/DigitalSubscription.scala

```diff
@@ -67,20 +67,16 @@ class DigitalSubscription(
 
   def digitalGeoRedirect: Action[AnyContent] = geoRedirect("subscribe/digital")
 
-  def displayForm(countryCode: String, displayCheckout: Boolean, isCsrf: Boolean = false): Action[AnyContent] =
+  def displayForm(countryCode: String): Action[AnyContent] =
```

Comment:
what was `isCsrf` doing before?

Reply:
I'm not sure - when I checked it was unused so I just tidied it up. Perhaps @rupertbates will know why it existed?